babble/src/main/java/io/mosaicnetworks/babble/utils/DialogUtils.java

```diff
@@ -64,10 +64,6 @@ public static void displayOkAlertDialogHTML(Context context, @StringRes int titl
                 .setNeutralButton(R.string.ok_button, null)
                 .create();
 
-        String fullhtml = "<?xml version=\"1.0\" encoding=\"UTF-8\" ?>\n" +
-                "<html><head><meta http-equiv=\"Content-Type\" content=\"text/html; charset=utf-8\"></head><body>\n" +
```

Comment:
Any reason for chopping this? 

It was recommended in StackOverflow to help with some edge cases. I think I had a device that did not work without it. Can't remember which one though.

Reply:
It's not used?

Reply:
That is a good point.

As you were.